app/api-v1/controllers/Recipe/index.js

```diff
@@ -40,10 +37,12 @@ module.exports = {
       const { id } = req.params
       if (!id) throw new BadRequestError('missing params')
 
-      const [recipe] = await db.client.from('recipes').select('*').where({ id })
+      const [recipe] = await db.getRecipe(id)
       if (!recipe) throw new NotFoundError('recipes')
 
       const payload = {
+        file: recipe.binary_blob,
```

Comment:
I think it's a little confusing for the image to only be called `file` since `requiredCerts` is also a file

Reply:
indeed!

Reply:
that has been changed to image for time being, since this is something that will be used by other controllers as well so later we can abstract or make it something generic